commit 483082387cbb40561aeb6ad566d2aa247318f817
Merge: 8b967a9 523c7ea
Author: Scott Lahteine <thinkyhead@users.noreply.github.com>
Date:   Sat Oct 22 13:54:09 2016 -0500

    Merge pull request #5022 from esenapaj/Fix-for-the-PR-#4553
    
    Fix for the PR #4553 (Save bytes for custom chars (Hitachi LCD))

diff --git a/Marlin/ultralcd_impl_HD44780.h b/Marlin/ultralcd_impl_HD44780.h
index 7c0a4c5..d43f88e 100644
--- a/Marlin/ultralcd_impl_HD44780.h
+++ b/Marlin/ultralcd_impl_HD44780.h
@@ -224,6 +224,16 @@ static void lcd_set_custom_characters(
     B10001,
     B01110
   };
+  static byte uplevel[8] = {
+    B00100,
+    B01110,
+    B11111,
+    B00100,
+    B11100,
+    B00000,
+    B00000,
+    B00000
+  }; //thanks joris
   static byte feedrate[8] = {
     B11100,
     B10000,
@@ -252,16 +262,6 @@ static void lcd_set_custom_characters(
   lcd.createChar(LCD_STR_CLOCK[0], clock);
 
   #if ENABLED(SDSUPPORT)
-    static byte uplevel[8] = {
-      B00100,
-      B01110,
-      B11111,
-      B00100,
-      B11100,
-      B00000,
-      B00000,
-      B00000
-    }; //thanks joris
     static byte refresh[8] = {
       B00000,
       B00110,
@@ -330,6 +330,8 @@ static void lcd_set_custom_characters(
       lcd.createChar(LCD_STR_FOLDER[0], folder);
     #endif
 
+  #else
+    lcd.createChar(LCD_STR_UPLEVEL[0], uplevel);
   #endif
 }
 
